tokamak_example.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
`tokamak_example.py` could go into `examples/tokamak/`?

Reply:
Moved into `examples/tokamak`, and changed to use yaml input files.